lib/components/side_menu.dart

```diff
@@ -1,50 +1,93 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:project_eureka_flutter/models/user_model.dart';
 import 'package:project_eureka_flutter/screens/home_page.dart';
 import 'package:project_eureka_flutter/screens/new_question_screens/new_question_screen.dart';
 import 'package:project_eureka_flutter/screens/profile_screen.dart';
 import 'package:project_eureka_flutter/screens/settings_screen.dart';
+import 'package:project_eureka_flutter/services/users_service.dart';
+import 'package:project_eureka_flutter/services/email_auth.dart';
 
-class SideMenu extends StatelessWidget {
+class SideMenu extends StatefulWidget {
   final String title;
-  SideMenu({Key key, this.title}) : super(key: key);
+  SideMenu({
+    this.title,
+  });
+
+  @override
+  _SideMenuState createState() => _SideMenuState();
+}
+
+class _SideMenuState extends State<SideMenu> {
+  UserModel user = new UserModel(
+    firstName: '',
+    lastName: '',
+    email: '',
+    pictureUrl: '',
+  );
+
```

Comment:
Where do I go to change my profile picture? Can you add a button where I can add a profile picture?

Reply:
That's an unrelated task to the side menu and it's currently in Open in our JIRA board.